### PR TITLE
Make extensions documentation build scripts prefer README.md if present

### PR DESCRIPTION
**Pull request by britice**
Addresses issue https://github.com/ampproject/docs/issues/20.

### CODE REVIEW

File: _scripts/import_reference_docs.js [hunk 1]
```diff
@@ -152,28 +152,44 @@ ghrepo.contents('extensions', "master", function(err, data) {
...
+				//   backwards compatibility with an older file structure in the
+				//   'extensions' directory. Rather than a `README.md`, each extension
+				//   had a `.md` file with the extension name (e.g. 'amp-font.md') which
+				//   was meant to be used as the README for that extension. When all
+				//   extension READMEs have been changed to `README.md`, this loop can
+				//   be simplified.
 				if(data[i].type === 'file' && /\.md$/.test(data[i].name)) {
-					component = data[i];
-					break;
+					if (!componentReadme) {
```
Comment (britice) [inline _scripts/import_reference_docs.js]: Capturing the first `.md` in the component matches the previous import convention. The new naming convention for primary readmes of extension components is `README.md` (e.g. `amp-font/amp-font.md` -> `amp-font/README.md`).

**Comment by pbakaus** [comment]
Is this PR still relevant? If so, could you resolve the conflicts?

**Comment by Meggin** [comment]
+1 to Paul's comment. We can't merge until the conflicts are resolved.

**Comment by pbakaus** [comment]
Closing due to inactivity. Please reopen if still relevant.

**Pull request closed without merging**